### PR TITLE
EDITORIAL CLOSING: XMPP Newsletter September 2021

### PR DESCRIPTION
**Pull request by Echolon**
I created the draft for the next release covering **September 2021**. 

You can make a simple comment or create a merge request to make things easy going for review. You don't need any special technical knowledge to contribute. If you need help, please ask for help in the Communication-Team MUC: commteam@muc.xmpp.org

- [You can use our online Pad](https://yopad.eu/p/xmpp-newsletter-365days)! Please be tidy and write something instead of dropping bare HTTP links :pray: 
- Just paste your prepared text in this thread below or in our XMPP channel (xmpp:commteam@muc.xmpp.org?join)
- [Make a pull request by editing the file in this repository](https://github.com/xsf/xmpp.org/blob/newsletter-august2021/content/posts/newsletter/2021-09-05.md)

:arrow_right: Please do not just drop bare links - rather try to make a short and decent formulated description (**<5 lines**) what is the main outcome of your source. You do not need to write in a hard, formal tone - be passionate why this motivates you or what you work on! 

**Note: If you create a pull request, which is highly appreciated, please @mention me once your PR is created!** :pray: 

XMPP Newsletter Timeline

---

This is our planed timeline for this draft:

Status:
- [x] Collection of news
- [x] Review
- [ ] Blog post + Tiny Newsletter
- [ ] Social media
- [ ] Translations
- [ ] Archived ([Use pandoc!](https://pandoc.org))

### CODE REVIEW

**Comment by wurstsalat3000** [comment]
I updated this PR to the new Hugo page structure and frontmatter.

File: content/blog/2021-10-05.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+## Events
+
+[XMPP Office Hours](https://wiki.xmpp.org/web/XMPP_Office_Hours) - Also, checkout our new [YouTube channel](https://www.youtube.com/channel/UCf3Kq2ElJDFQhYDdjn18RuA)!
+
+[Berlin XMPP Meetup (remote)](https://mov.im/?node/pubsub.movim.eu/berlin-xmpp-meetup): Monthly Meeting of XMPP Enthusiasts in Berlin - always 2nd Wednesday of the month.
+
+## Articles
+
+OpenPGP for XMPP (OX) is slowly getting client implementations, in a German blog post DebXWoody takes us step by step in [enabling it and using it in Profanity](https://mov.im/?blog/debxwoody%40movim.eu/profanity-und-openpgp-for-xmpp-ox-YRMREh).
```
Comment (wurstsalat3000) [inline content/blog/2021-10-05.md:41]: implementations. In a German blog post, DebXWoody walks us step by step through the process of enabling OX and using it in Profanity

File: content/blog/2021-10-05.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+[XMPP-DNS](https://salsa.debian.org/mdosch/xmpp-dns), a tool to look up XMPP SRV records and test connectivity, had its initial release of [v0.1.0](https://salsa.debian.org/mdosch/xmpp-dns/-/releases/v0.1.0) directly followed by the release of [v0.2.0](https://salsa.debian.org/mdosch/xmpp-dns/-/releases/v0.2.0) bringing support for xmpps-server SRV records and a small bugfix release [v0.2.1](https://salsa.debian.org/mdosch/xmpp-dns/-/releases/v0.2.1).
```
Comment (wurstsalat3000) [inline content/blog/2021-10-05.md:55]: v0.1.0. The release was directly followerd by [...]

File: content/blog/2021-10-05.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+
+Conversations and Quicksy got version [2.10.0](https://github.com/iNPUTmice/Conversations/releases/tag/2.10.0) out this month, with a short changelog: black bars on video calls (so you know when "you're holding it wrong"), search performance improvements and a new setting to block app screenshots. Under the hood there was more: two bugs fixed for file attachments (specially for users with a lot of media files), touching the titlebar will open chat details and nested quotes (not yet the default, but you can "copy" and then "paste as quote" to use it).
+
+Converse is going forward after a lot of development, [version 8 of this JavaScript XMPP chat client that runs in your browser](https://opkode.com/blog/2021-09-13-converse-8/) was released. [JC Brand](https://opkode.com)'s blog post covers the visible changes (message styling, OMEMO encrypted
```
Comment (wurstsalat3000) [inline content/blog/2021-10-05.md:64]: after a lot of development. Version 8 of [...]

File: content/blog/2021-10-05.md [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+
+Image https://conversejs.org/screenshots/Converse-fullscreen.png
+
+Profanity [0.11.1](https://github.com/profanity-im/profanity/releases/tag/0.11.1) has been released improving upon themes, notifications and OMEMO hondling.
+
+The Mellium Dev Communiqué for September has been published. It includes minor updates to the [Communiqué TUI client](https://mellium.im/communique/) as well as the [mellium.im/xmpp](https://pkg.go.dev/mellium.im/xmpp) library. Full details on their [Open Collective](https://opencollective.com/mellium) page: https://opencollective.com/mellium/updates/dev-communique-for-september-2021
+
+## Servers
+
+No news on XMPP servers has reach us this month. :-(
```
Comment (wurstsalat3000) [inline content/blog/2021-10-05.md:75]: have reached us

**Review by wurstsalat3000 — COMMENTED** [review]
@Echolon

File: content/blog/2021-10-05.md (hunk 1 above)
Comment (Echolon) [reply content/blog/2021-10-05.md:41]: @wurstsalat3000 many thanks - can you push these? I'm not home till sunday

**Comment by Echolon** [comment]
:hourglass: **Editorial closing of the XMPP Newsletter September 2021 release**
:rocket: Merge in a few minutes! Docker build pulls it asap robot
:pray: Thanks to Adrien Bourmault (neox), Benoît Sibaud, emus, palm123, Licaon_Kter, MattJ, mdosch, nicola, seveso, Sam Whited, SouL, wurstsalat3000, Ysabeau for their help in creation, review and translation!
:handshake: ... ehh.. here we go again, https://timetoherd.com/ hand mask hand
:heart: :fireworks: Next release on 5th of November 2021 - The upcoming last release for this year!

**Pull request merged**